crates/mlsirm-core/src/longitudinal_irt.rs

```
@@ -0,0 +1,1733 @@
+        observed_count,
+        transition_count,
+        status,
+        estimand_scope: ESTIMAND_SCOPE,
+        transition_kind: TRANSITION_KIND,
+        interval_kind: INTERVAL_KIND,
+        engine: ENGINE,
+    })
+}
+
+/// Deterministic simulator for hierarchical CT-AR Rasch recovery fixtures.
+pub fn simulate_hierarchical_ctar_rasch(
```

Clippy (1.97.1)

[warning] this function has too many arguments (8/7)

(warning)